src/entities/addLiquidityBoosted/index.ts

```diff
@@ -85,6 +87,7 @@ export class AddLiquidityBoostedV3 {
                     input.sender ?? zeroAddress,
                     input.userData ?? '0x',
                     poolState.address,
+                    input.wrapUnderlying,
                     maxAmountsIn,
```

Comment:
I believe this `maxAmountsIn` is relying on `childTokens`, which needs to be updated here for add liquidity unbalanced (similar to what you did on the add liquidity proportional case)

Reply:
nice catch! :raised_hands: